src/kili/adapters/kili_api_gateway/notification/operations.py

```diff
@@ -0,0 +1,18 @@
+"""Collection of notification's related GraphQL queries and mutations."""
```

Comment:
How about changing that to "Collection of GraphQL queries and mutations that are related to the notification"?